server/channels/store/localcachelayer/layer_test.go

```diff
@@ -148,3 +150,28 @@ func tearDownStores() {
 		wg.Wait()
 	})
 }
+
+func TestClearCacheCluster(t *testing.T) {
+	cluster := &testlib.FakeClusterInterface{}
+	lc := &LocalCacheStore{
+		cluster: cluster,
+	}
+
+	c := cache.NewLRU(&cache.CacheOptions{
+		Size:                   10,
+		Name:                   "test",
+		InvalidateClusterEvent: model.ClusterEventInvalidateCacheForRoles,
+	})
+
+	lc.doClearCacheCluster(c)
+	assert.Len(t, cluster.GetMessages(), 1)
```

Comment:
Could we do an equality match on the cluster message to assert it's what we expect?

Reply:
Done